### DISCUSSION TITLE
[@types/jsonld] toRDF return type should be Promise<string>

### DISCUSSION THREAD

**Discussion by adlerfaulkner** [DefinitelyTyped/DefinitelyTyped#60661] [Issues with a @types package]
According to [the digitalbazaar/jsonld README](https://github.com/digitalbazaar/jsonld.js#tordf-n-quads), the `toRDF` function returns a `string` of N-Quads not `type RdfDataSet = object;`

This line:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f7ec78508c6797e42f87a4390735bc2c650a1bfd/types/jsonld/index.d.ts#L143

Needs to change to 
```
export function toRDF(input: JsonLdDocument, options?: Options.ToRdf): Promise<string>;
```

[a comment by an automated account is not shown]

**Comment by agustaf9** [comment]
It looks like it was changed here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41978
The change was made based on this function: https://github.com/digitalbazaar/jsonld.js/blob/5367858d28b6200aaf832d93eb666d4b819d5d4f/lib/toRdf.js#L53
which returns an RDF dataset, when I believe it should have been based on this function: https://github.com/digitalbazaar/jsonld.js/blob/5367858d28b6200aaf832d93eb666d4b819d5d4f/lib/jsonld.js#L668
Which can return a string.
